### PR TITLE
add search reporter lambda

### PR DESCRIPTION
**Pull request by harrisonpim**

### CODE REVIEW

**Comment by kenoir** [comment]
Nice. Excited to see this in action.

File: search_reporter/lambda/src/elastic.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        body={
+            "query": {
+                "range": {
+                    "timestamp": {
+                        "gte": start,
+                        "lt": end
+                    }
+                }
+            },
+            "size": 100_000
```
Comment (jamesgorrie) [inline search_reporter/lambda/src/elastic.py:43]: What's this?

Comment (harrisonpim) [reply search_reporter/lambda/src/elastic.py:43]: increasing the upper limit on how many documents are returned. the default (10,000 docs) is sometimes not enough to cover the full time window

Comment (jamesgorrie) [reply search_reporter/lambda/src/elastic.py:43]: I didn't know [this](https://www.python.org/dev/peps/pep-0515/) was a thing.

Comment (harrisonpim) [reply search_reporter/lambda/src/elastic.py:43]: Oh yeah, it's a bit weird and ugly and old school but i've fallen into a habit of using it for numbers with lots of 0s. Too easy to confuse one million / one hundred thousand etc

Comment (jamesgorrie) [reply search_reporter/lambda/src/elastic.py:43]: I quite like it.

**Review by jamesgorrie — APPROVED** [review]
Added the terraform remote state, we can break some tidy up tickets out and work on those.

**Pull request merged**